### DISCUSSION TITLE
Update is changing createdAt

### DISCUSSION THREAD

**Discussion by cfgeek** [wheels-dev/wheels#1274] [Q&A] [answered]
Hi All,

I have a cfwheels 2.0.1 app running which the client has reached out to let me know the createdAt is changing when they update a entry.

I have confirmed that it is not the display or anything like that and the database is having the createdAt column updated when executing an update of a record. I went ahead and jumped into the wheels\model\update.cfm file and changed "UPDATE" to "UPDATES" on line 236 in order to make sure it errors out and I can see the SQL in the error. I have pasted it below. The thing I was looking to confirm is that the SQL is indeed updating the createdAt value in an update request. Does anyone have a clue why it would be doing this and how I can fix it. Also running on Lucee 5.3.8.206.

<html>
<body>

Message | You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'UPDATES jobs SET createdat ='2023-08-01 19:58:00.0',updat' at line 1
-- | --
SQL | UPDATES jobs SET createdat ={ts '2023-08-01 19:58:00'}, updatedat ={ts '2023-08-01 20:03:21'} WHERE id =15991

</body>
</html>

**Comment by neokoenig** [comment, marked as answer]
I believe this has been fixed in later versions - https://github.com/cfwheels/cfwheels/issues/927

**Reply by cfgeek** [reply]
Sounds like I need to get the client to let me update wheels then :-)
